pkg/kubelet/token/token_manager.go

```diff
@@ -38,13 +40,14 @@ const (
 )
 
 // NewManager returns a new token manager.
-func NewManager(c clientset.Interface) *Manager {
+func NewManager(clients []clientset.Interface) *Manager {
```

Comment:
Considering using map tpId to clientset. It will be much easier for later dynamic TP/RP support

Reply:
as discussed offline, this change is to fix the defect that token_manager unable to support multiple TPs in scale-out env.